content/doc/book/pipeline/pipeline-best-practices.adoc

```diff
@@ -0,0 +1,97 @@
+---
+layout: section
+title: Pipeline Best Practices
+---
+ifdef::backend-html5[]
+:notitle:
+:description:
+:author: Alex Taylor
+:email: jenkinsci-docs@googlegroups.com
+:sectanchors:
+ifdef::env-github[:imagesdir: ../resources]
+ifndef::env-github[:imagesdir: ../../resources]
+:hide-uri-scheme:
+:toc:
+endif::[]
+
+= Pipeline Best Practices
+
+== General
+
+=== Pipeline functions as Glue
```

Comment:
What is a Pipeline function? If you mean "everything but steps", I would maybe say "Pipeline scripts" or "Groovy code in Pipelines".